src/R/Components/Impl/Application/Configuration/IO/ConfigurationSettingsWriter.cs

```diff
@@ -64,16 +66,16 @@ public sealed class ConfigurationSettingsWriter : IConfigurationSettingsWriter {
         }
 
         private static string FormatValue(IConfigurationSetting s) {
```

Comment:
While you are at it, could you fix ##2420? If category is SQL, we should do `\\`